docs/json/radarr/cf/truehd-atmos.json

```diff
@@ -23,7 +23,7 @@
       "negate": false,
       "required": true,
       "fields": {
-        "value": "\\bATMOS(\\b|\\d)"
+        "value": "\\b(ATMOS|W4NK3R|-DON)(\\b|\\d)"
```

Comment:
wouldn't `HQMUX` be needed here?

Reply:
Again, restored this line to it's previous state. This CF needs both `TrueHD` and `ATMOS` conditions to match. So, if a release name has both `TrueHD` and `-DON` in the title then the CF will match, which is what we want to achieve.